include/cutlass/gemm/kernel/xe_gemm.hpp

```diff
@@ -162,11 +162,28 @@ class GemmUniversal<
     auto m = get<0>(args.problem_shape);
     auto n = get<1>(args.problem_shape);
     auto k = get<2>(args.problem_shape);
+    auto l = get<3>(args.problem_shape);
+    bool is_batch = l > 1;
+    auto check_stride = [is_batch](auto stride, int el_size){
+      auto a = get<0>(stride);
+      auto b = get<1>(stride);
+      auto valid_is_unit = a == _1{} || b == _1{};
+      auto inner = a == _1{} ? b : a;
+      auto valid_inner = inner % (16 / el_size) == 0;
+      auto valid_outer = !is_batch || get<2>(stride) % (64 / el_size) == 0;
```

Comment:
Do we want to test:
> the Memory Width is greater than or equal to 64 bytes

?

Reply:
I can not check on BMG right now, but IIRC, it was working correctly there even if that was not the case. I did check on PVC.

Reply:
Checked it on BMG, works fine with e.g. `--m=8 --k=8` :+1: